ui/components/imageWorkspace/src/components/Features/TextInputFeature.svelte

```diff
@@ -43,12 +55,21 @@
   {#if isEditing}
     <Input
       value={textFeature.value}
-      on:change={(e) => onTextInputChange(e.currentTarget.value, textFeature.name)}
-      on:input={() => (isSaved = false)}
       type={inputType === "str" ? "text" : "number"}
       step={inputType === "int" ? "1" : "any"}
+      list="scene_availableValues_{textFeature.name}"
+      on:change={(e) => onTextInputChange(e.currentTarget.value, textFeature.name)}
+      on:input={() => (isSaved = false)}
       on:keyup={(e) => e.stopPropagation()}
     />
+    <datalist id="scene_availableValues_{textFeature.name}">
```

Comment:
there is a `Combobox` component available (only available with search feature for now but could be easily improved). 
You can find documentation here : https://www.shadcn-svelte.com/docs/components/combobox 

I can help on that if you want?